src/test/scala/com/microsoft/hyperspace/index/E2EHyperspaceRulesTests.scala

```diff
@@ -450,7 +450,64 @@ class E2EHyperspaceRulesTests extends HyperspaceSuite with SQLHelper {
 
       // Verify index usage on latest version of index (v=1) after refresh.
       verifyIndexUsage(query2, getIndexFilesPath(indexConfig.indexName, 1))
-      FileUtils.delete(dataPath)
+      FileUtils.delete(new Path(location))
```

Comment:
Could you move this to `after()` or use a different path for "location" in the below test?
If this test failed, test data might affect the below test.

Reply:
Sure, I changed the path used in tests to avoid any side-effect due to failures.